time timestamp
float64[] posx
float64[] posy
float64 robotPosx
float64 robotPosy
int64[] axisID
int64[] mobileID
float64 vx
float64 vy
float64 theta
uint8 state
int64[] active_nodes_l
int64[] active_nodes_r
float64[] pK
float64[] node_x
float64[] node_y
float64[] node_id
float64[] R
